public/src/components/channelManagement/helpers/shared.ts

```diff
@@ -5,6 +5,8 @@ export interface Variant {
   name: string;
 }
 
+export type TestPlatform = 'AMP' | 'DOTCOM';
```

Comment:
The apple news epic tool also needs validation - it only supports `CURRENCY_SYMBOL` - https://github.com/guardian/apple-news/blob/73bf7b48d02b721425e9b80b3c17a222f8d0a288/app/integration/EpicService.scala